ext/openssl/ossl_digest.c

```diff
@@ -414,6 +440,7 @@ Init_ossl_digest(void)
 
     rb_define_method(cDigest, "initialize", ossl_digest_initialize, -1);
     rb_define_method(cDigest, "initialize_copy", ossl_digest_copy, 1);
+    rb_define_module_function(cDigest, "digests", ossl_s_digests, 0);
```

Comment:
This is a little unusual place for a class method. Please move it to above other instance method definitions.